src/Symfony/Component/Filesystem/Tests/FilesystemTest.php

```diff
@@ -605,6 +605,10 @@ public function testChgrpSymlink()
      */
     public function testChgrpSymlinkFails()
     {
+        if (defined('HHVM_VERSION')) {
+            $this->markTestSkipped('Shouldn\'t be executed in HHVM context.');
```

Comment:
See https://github.com/facebook/hhvm/commit/8358b899bfd6d224fb85b08d7a2620172bf0fc7a this will be fixed in the next nightly version.